sql/ll_bitemporal_update.sql

```diff
@@ -99,7 +99,7 @@ into v_keys;
                                   
 EXECUTE 
 --v_sql :=
-format($u$ UPDATE %s SET (%s) = ( %s ) 
+format($u$ UPDATE %s SET (%s) = (select %s ) 
```

Comment:
what is this?

Reply:
this was a change i made, which is actually  an error, so I'm fixing it. 
This is how it is in the OG fork